sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -197,7 +205,7 @@ void viewvideo_init(void)
 
 #if VIEWVIDEO_USE_NETCAT
   // Create an Netcat receiver file for the streaming
-  sprintf(save_name, "%s/netcat-recv.sh", STRINGIFY(VIEWVIDEO_SHOT_PATH));
```

Comment:
why remove `STRINGIFY` here? Now you need to make sure it is properly quoted if you want to define this yourself....